crates/orchestrator/src/tests/jobs/mod.rs

```diff
@@ -477,8 +491,12 @@ async fn verify_job_with_rejected_status_works() {
     sleep(Duration::from_secs(5)).await;
 
     // Queue checks.
-    let consumed_messages_processing_queue =
-        services.config.queue().consume_message_from_queue(JOB_PROCESSING_QUEUE.to_string()).await.unwrap_err();
+    let consumed_messages_processing_queue = services
+        .config
+        .queue()
+        .consume_message_from_queue(DATA_SUBMISSION_JOB_PROCESSING_QUEUE.to_string())
```

Comment:
Any reason as to why we have not used job_item.job_type.process_queue_name() function here and similar places ? What am I missing ?
@byteZorvin